Newsletters: add 204 (2022-06-15)

harding opened the pull request:
- [x] Add lede, update releases/RCs, add topic links @harding 
- [x] Bitcoin 24171 sync chain from inbound peers @Xekyo 
- [x] LDK 1505, BDK 628, and BDK 593 @harding 

(Sorry for being a bit late this week.)

adamjonas commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,281 @@
+      negotiation between two peers to support package relay should
+      include information about each peer's package maximum size and
+      depth limits, otherwise nodes with non-default settings could
+      receive repeated notifications about packages they were did not
```
```suggestion
      receive repeated notifications about packages they did not
```

adamjonas commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,281 @@
+      depth limits, otherwise nodes with non-default settings could
+      receive repeated notifications about packages they were did not
+      want, wasting bandwidth.  BIP author Gloria Zhao [suggests][zhao
+      negotiation] using that the first version of the package relay
```
```suggestion
      negotiation] using the first version of the package relay
```

adamjonas commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,281 @@
+
+    - *Non-standard parents:* Suhas Daftuar [describes][daftuar repeat]
+      a scenario where a node implementing package relay can end up
+      repeatedly requesting the same package.  This is would be
```
```suggestion
      repeatedly requesting the same package.  This would be
```

adamjonas commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,281 @@
+
+    - *Challenges of a block hash beacon:* Daftuar also notes that one
+      feature of the proposal may cause problems for other software.
+      The current draft BIP has packages include the node's current hash
```
```suggestion
      The current draft BIP has packages that include the node's current hash
```

adamjonas commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,281 @@
+      signature using both her hot wallet and a hardware signing
+      device.  It was discussed how to design LN's use of MuSig2 to
+      ensure recursive MuSig2 was available.  Also the security of
+      recursive of MuSig2 was discussed.
```
```suggestion
      recursive MuSig2 was discussed.
```

Xekyo replied:
I haven't heard of "recursive MuSig2" before. How is that different from other ways of constructing an aggregated public key and signatures using MuSig?

harding replied:
Adding details.  Thanks!

adamjonas commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,281 @@
+      Additionally, one previously-discussed thought experiment is to
+      take the information that a probing node would learn and have
+      nodes share it freely without requiring any probing.  If that was
+      done by every node, it's bandwidth requirements and loss of
```
```suggestion
      done by every node, its bandwidth requirements and loss of
```

adamjonas commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,281 @@
+      [BOLT11][] invoice from a webserver; the BOLT12 [offers][topic
+      offers] protocol allows requesting an invoice from a node on the
+      network.  Among other aspects of these protocols, participants
+      discussed how their could be made compatible with each other so
```
```suggestion
      discussed how they could be made compatible with each other so
```

Xekyo commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,286 @@
+      sids] that short ids make sense for block relay where nodes first
+      validate a new block's proof of work (which is expensive to
+      create), so it would be expensive for an attacker to abuse the
+      mechanism to waste node resources.  But, for relay where no
+      significant expense is involved, short ids can be abused over and
```
"where no significant expense is involved could perhaps be a bit more specific. How about: 

```suggestion
      mechanism to waste node resources.  But, for relay of data that is not 
      expensive to create, short ids can be abused over and
```

Xekyo commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,286 @@
+      [Newsletter #164][news164 taproot ln]).
+
+    - *Tapscript and MuSig2:* as part of the switch to taproot-based
+      channels, there is a need to covert existing scripts to tapscript
```
```suggestion
      channels, there is a need to convert existing scripts to tapscript
```

Xekyo commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,286 @@
+  [detailed summary][osuntokun summary] of the LN dev meeting in Oakland
+  last week.  Topics covered included:
+
+    - *Taproot-based LN channels:* the first step of moving LN to full
+      use of [taproot's][topic taproot] features, with later steps
+      likely to include support for [PTLCs][topic ptlc] (see also
+      [Newsletter #164][news164 taproot ln]).
```
This paragraph feels incomplete. Is the first step of moving LN to Taproot taproot-based channels or is this just a setup for the next paragraphs? I assume this means using a P2TR output in the commitment transaction. Perhaps that could be clarified.

Xekyo commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,286 @@
+      preventing any intermediate node from learning the network
+      identity of either the spender or receiver.  This outsourcing is
+      especially useful for mobile LN clients who aren't attempting to
+      route other node's payments.  As mentioned in the meeting summary,
```
```suggestion
      route other nodes' payments.  As mentioned in the meeting summary,
```

Xekyo reviewed (commented):
I love the comprehensive LN summit notes. Don't know how I'd keep up with Lightning if you didn't write up the main gist for us. :)

harding commented:
Added remaining merge summaries, updated RCs, added topic links, and reviewed @Xekyo's commit (thanks!).  Also addressed all review comments, thanks @adamjonas @Xekyo and @glozow!

nechteme commented on _posts/en/newsletters/2022-06-15-newsletter.md:
```diff
@@ -0,0 +1,294 @@
+      MuSig2 would allow, for example, Alice to create her part of the
+      signature using both her hot wallet and a hardware signing
+      device without Bob performing any special steps or even knowing
+      that Alice was signinign with more than one key.
```
```suggestion
      that Alice was signing with more than one key.
```

bitschmidty commented:
Pushed a small spelling commit (https://github.com/bitcoinops/bitcoinops.github.io/pull/782#discussion_r897512471),  squashed, and merged! 

Thank you @harding @Xekyo @adamjonas @nechteme !

MaxHillebrand commented:
A heads up, we are about to release Wasabi Wallet 2.0, the mainnet release.

WasabiWallet.io
Github.com/zkSNACKs/WalletWasabi

Sorry for the very short notice.
